test/gateways/trexle_test.exs

```diff
@@ -62,6 +62,70 @@ defmodule Gringotts.Gateways.TrexleTest do
         assert response.status_code == 400
       end
     end
+
+    test "test_for_purchase_with_valid_amount" do
+
+    end
+
+    test "test_for_purchase_with_invalid_amount" do
+      
+    end
   end 
 
+  describe "authorize" do
+
+    test "test_for_authorize_with_valid_card" do
```

Comment:
@jyotigautam remove underscores from the statements, 

Ultimately, the out would be `test authorize test_for_authorise_with_valid_card` 

make sure this is consistent with `test authorise ...(meaningful statement)...`